### PR TITLE
update swagger.yaml

### PR DESCRIPTION
**Pull request by AaronLasseigne**
Updating swagger.yaml based on changes to mode/webapp.

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -2033,8 +2033,8 @@ paths:
     get:
       operationId: listCollections
       summary: List Collections
-      description: Returns a representation of `Collection`s in a `Workspace` visible to
-        the authenticated `User`
+      description: Returns a representation of `Collections`s in a `Workspace` visible
```
Comment (jrgarcia) [inline swagger.yaml]: I'm guessing this shouldn't be `'Collections's` (note the double "s").

File: swagger.yaml [hunk 2]
```diff
@@ -2049,6 +2049,14 @@ paths:
           schema:
             "$ref": "#/definitions/Spaces"
       parameters:
+      - in: query
+        name: filter
+        description: 'Returns a filtered list of `Collection`s. Valid options include
```
Comment (jrgarcia) [inline swagger.yaml:2054]: You should be able to remove the outside quotes for easier reading (and not requiring the double `'`).

**Review by jrgarcia — CHANGES_REQUESTED** [review]
Open questions with some minor nitpicks.

File: swagger.yaml (hunk 2 above)
Comment (AaronLasseigne) [reply swagger.yaml:2054]: This is autogenerated by a rake task that calls `to_yaml`. I guess that's just how it handles this situation.

File: swagger.yaml (hunk 1 above)
Comment (AaronLasseigne) [reply swagger.yaml]: PR to fix in webapp: https://github.com/mode/webapp/pull/15558

**Review by msalli — APPROVED** [review]
👍

File: swagger.yaml (hunk 2 above)
Comment (jrgarcia) [reply swagger.yaml:2054]: Ah, ok. That makes sense.

**Pull request merged**